apps/src/gamelab/spritelab/commands.js

```diff
@@ -30,7 +30,6 @@ function updateTitle() {
 
 export const commands = {
   executeDrawLoopAndCallbacks() {
-    this.createEdgeSprites();
```

Comment:
Nothing else (besides the two commands above) depend on the edge sprites in Spritelab?

Reply:
Nope

Reply:
Update: we implicitly depended on `edges` being a global group in some validation code: https://github.com/code-dot-org/code-dot-org/pull/29607#discussion_r302319245.